frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -197,6 +197,32 @@ export default class AccountsApi extends ExplorerApi {
     }
   }
 
+  async queryDbAccount(accountId: string): Promise<any> {
+    try {
+      let accountDetails;
+      if (this.dataSource === DATA_SOURCE_TYPE.INDEXER_BACKEND) {
+        accountDetails = this.call<any>("select:INDEXER_BACKEND", [
```

Comment:
The returned value is not `accountDetails`. Please, be mindful of variable naming since misleading naming is worse than random names.